packages/util/CHANGELOG.md

```diff
@@ -6,6 +6,100 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 9.0.0-rc.1 - 2023-07-11
```

Comment:
Ok, will add a note. Just to note though as well that we *didn't* have these methods *at all* before in the last released version, so this can't confuse existing code.

Reply:
Update: ah, just re-reading the Util release notes: I have really intended this to be "forwarded" to the Util README being the main entrypoint for this kind of information, and the Util CHANGELOG has the following section as well:

```shell
We have converted existing Buffer conversion methods to Uint8Array conversion methods in the [@ethereumjs/util](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/util) `bytes` module (so: within this library), see the respective README section for guidance.
```

Would therefore keep this "as is", this would rather duplicate things otherwise.